codechain/subcommand/mod.rs

```diff
@@ -27,6 +27,10 @@ pub fn run_subcommand(matches: &ArgMatches) -> Result<(), String> {
     match subcommand.name.as_str() {
         "account" => run_account_command(&subcommand.matches),
         "convert" => run_convert_command(&subcommand.matches),
+        "commitHash" => {
```

Comment:
How about `commit-hash`?
I'm familiar with it rather than the pascal case for cli options.

Reply:
I changed it.